components/authz-service/server/subject_purge_test.go

```diff
@@ -39,7 +39,7 @@ func TestCommonSubjectPurgeService(t *testing.T) {
 
 	properties.Property("calls both with subject, returns output", prop.ForAll(
 		func(sub string, v2Pols []string) bool {
-			s, err := v2.NewSubjectPurgeServer(ctx, l,
+			s, err := server.NewSubjectPurgeServer(ctx, l,
```

Comment:
we don't need this as a separate server any longer right? let's get a card for merging it into the policy server

Reply:
Trueeeee! Good catch.